packages/schema/bind/src/bindings/wasm-rs/transforms/detectCircular.ts

```diff
@@ -0,0 +1,5 @@
+// TODO:
```

Comment:
Just leaving a comment here to make sure this doesn't end up being forgotten
Or, if it's intended as a separate PR, then make sure to make an issue to track it